gluster-block.toml

```diff
@@ -0,0 +1,5 @@
+GlusterBlockCLIPath = "/usr/bin/glusterd"
```

Comment:
Not matching `toml` field name

Reply:
Default conf file(`$SRC/extras/conf/config.toml.sample`) already exists and used in `main.go` can that be used?

Reply:
why config file is in `$SRC/extras/conf/config.toml.sample` it should be in source dir, right?

Reply:
You can move if required.

Reply:
`Makefile` needs change if moved.

Reply:
kept it in same folder .PTAL